src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -52,14 +53,14 @@ public int getSize() {
     }
 
     public int getIndex(K key) {
-        return key == null ? 0 : Math.abs(key.hashCode() % DEFAULT_CAPACITY);
+        return key == null ? 0 : Math.abs(key.hashCode() % table.length);
     }
 
     private void resize() {
         size = 0;
-        Node<K, V>[] newSize = table;
+        Node<K, V>[] tableCopy = table;
         table = new Node[table.length * 2];
-        for (Node<K, V> nodes : newSize) {
+        for (Node<K, V> nodes : tableCopy) {
             Node<K, V> node = nodes;
```

Comment:
please remove this variable - there is no need in it
check my previous comment once more

Reply:
Ok, this time I got you